addon/components/sessions-grid.js

```diff
@@ -64,6 +71,26 @@ export default Component.extend({
     this.set('confirmDeleteSessionIds', []);
   },
 
+  didInsertElement() {
+    this._super(...arguments);
+    this.scrollDown();
+  },
+
+  didRender() {
+    this._super(...arguments);
+    runTask(this, () => {
```

Comment:
`runTask` with `runDisposibles`  is the wrong API here. You want to use the event listener API at https://ember-lifeline.github.io/ember-lifeline/docs/add-event-listener

Reply:
I fixed it. I couldn't get `package-lock.json` to update still. Maybe I need help for that as I don't know what's going on.